qutebrowser/config/config.py

```diff
@@ -665,16 +674,21 @@ def _get_stylesheet(self) -> str:
         """
         return _render_stylesheet(self._stylesheet)
 
-    @pyqtSlot()
     def _update_stylesheet(self) -> None:
         """Update the stylesheet for obj."""
         self._obj.setStyleSheet(self._get_stylesheet())
 
+    @pyqtSlot(str)
+    def _update_stylesheet_conditional(self, option: str) -> None:
```

Comment:
For consistency with other similar methods, I'd call this `_maybe_update_stylesheet`.